custom/icds_reports/tasks.py

```diff
@@ -1199,6 +1199,46 @@ def build_incentive_report(agg_date=None):
         agg_date = current_month - relativedelta(months=1)
     for state in state_ids:
         AWWIncentiveReport.aggregate(state, agg_date)
+
+    awc_performance_rows_list = list(AWWIncentiveReport.objects.values_list('awc_id', 'is_launched',
+                                                                            'valid_visits', 'visit_denominator',
+                                                                            'wer_weighed', 'wer_eligible',
+                                                                            'incentive_eligible')
+                                     .order_by('awc_id'))
+    awc_aggregate_rows_list = list(AggAwc.objects.values_list('awc_id', 'is_launched',
```

Comment:
This has to be filtered out for aggregation_level=5

Reply:
since we are filtering with valid awc_ids, i think this can be left out